scripts/linter_utils.py

```diff
@@ -68,12 +73,17 @@ def get_from_cache(key, factory):
             *. The result of factory(), or the last value to be associated to
             key.
         """
-        if key in cache:
-            return cache[key]
         with threadsafe_access(key):
             if key not in cache:
-                cache[key] = factory()
-        return cache[key]
+                q = Queue.Queue()
```

Comment:
Don't just remove the comment, though -- I think you do still want to explain why we are doing all this stuff.